specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/kustoPool.json

```diff
@@ -0,0 +1,2393 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SynapseManagementClient",
+    "version": "2021-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Synapse/kustooperations": {
```

Comment:
>kustooperations [](http://example.com/codeflow?start=34&length=15)

These operations should be added under MS.Synapse/operations. No need to create separate operations lists